tests/Microsoft.Identity.Test.Unit/WebUITests/HttpListenerInterceptorTests.cs

```diff
@@ -90,31 +90,55 @@ await AssertException.TaskThrowsAsync<OperationCanceledException>(
         [TestMethod]
         public async Task ValidateHttpListenerRedirectUriAsync()
         {
-            HttpListenerInterceptor listenerInterceptor = new HttpListenerInterceptor(
-                Substitute.For<ILoggerAdapter>());
+            HttpListenerInterceptor listenerInterceptor = new(Substitute.For<ILoggerAdapter>());
 
             int port = FindFreeLocalhostPort();
+            listenerInterceptor.TestBeforeStart = (url) => Assert.AreEqual($"http://localhost:{port}/TestPath/", url);
```

Comment:
Should "http://localhost:" be defined as a constant? Surely it's being used in other tests, and in other files?

Reply:
same as the original code, the bug is not to fix the `Should "http://localhost/:" be defined as a constant`